### DISCUSSION TITLE
Creating the Repository for Zed

### DISCUSSION THREAD

**Discussion by 1chooo** [arcticicestudio/nord#261] [Suggestions & Ideas]
Recently, there's been a rise in popularity of the new Zed editor. While there are existing themes available for use, there hasn't been any sign of the Nord theme being developed yet. Therefore, I believe it's a good idea to create a new repository to start collaborating with developers who are interested in using the Nord theme in Zed!

**Comment by jcs090218** [comment]
I know there is an extension for Zed in https://github.com/mikasius/zed-nord-theme.
